cmd/cmount/arch.go

```diff
@@ -0,0 +1,7 @@
+package cmount
+
+// ProvidedBy returns true if the rclone build for the given
+// architecture provides support for lib/cgo-fuse
+func ProvidedBy(arch string) bool {
```

Comment:
I would prefer this to be moved to the cmount file in buildinfo so that one command doesn't depend on a function from another (selfupdated depending on this function from cmount).

Reply:
The function `ProvidedBy` is logically closer to `cmount`. Writing `cmount.ProvidedBy(arch)` feels more natural than `buildinfo.ProvidedBy(arch)`.
Though it lives in the `cmount` package, it is independent from the `cmount` command. The command may not be compiled in due to build conditions but the function will still be there.
Probably we could move it to `cmd/mountlib` and call as `mountlib.CmountProvidedBy`, but I am slightly biased to `cmount.ProvidedBy`.

Reply:
Moreover, the `lib/buildinfo/cmount.go` is not always present.

Reply:
Missed that. The current implementation is fine then.